src/webui/www/private/css/style.css

```diff
@@ -484,3 +484,17 @@ td.statusBarSeparator {
 .statisticsValue {
     text-align: right;
 }
+
+/* iOS iframe scrolling */
+.windowFrame .mochaContentWrapper {
+    overflow: auto !important;
```

Comment:
I played a bit, seems the `!important` is not strictly required here, no?

Reply:
In my testing there was an inline ‘overflow: hidden’ applied to the element, so ‘!important’ was required to get it to scroll on iOS. Were you able to test on an iOS device with ‘!important’ removed?

Reply:
Sadly I've no ios at hand. 
Maybe you can put this to the comment above, something like:
`!important is required for iOS iframe scrolling` ?

Reply:
I've added some comments to clarify the css. Regarding `!important`, the style hierarchy from the iOS dev tools:

<img width="1000" alt="screen shot 2018-03-06 at 10 45 37 pm" src="https://user-images.githubusercontent.com/8296030/37072791-3773cc78-2191-11e8-8e23-24a8b824981c.png">